kubectl-pg/cmd/scale.go

```diff
@@ -0,0 +1,91 @@
+// Copyright © 2019 NAME HERE <EMAIL ADDRESS>
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+	"strconv"
+	PostgresqlLister "github.com/zalando/postgres-operator/pkg/generated/clientset/versioned/typed/acid.zalan.do/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"strings"
+)
+
+// scaleCmd represents the scale command
+var scaleCmd = &cobra.Command{
+	Use:   "scale",
+	Short: "Scales postgresql object by cluster-name",
+	Long: `Scales the postgres objects using cluster-name.
+Scaling to 0 leads to down time.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		clusterName,_ := cmd.Flags().GetString("clusterName")
+		namespace,_ := cmd.Flags().GetString("namespace")
+		if len(args) > 0 {
+			numberofInstances,_ := strconv.Atoi(args[0])
+			scale(int32(numberofInstances),clusterName, namespace)
+		} else {
+			fmt.Println("Please enter number of instances to scale.")
+		}
+
+	},
+	Example: "kubectl pg scale [NUMBER-OF-INSTANCES] -c [CLUSTER-NAME] -n [NAMESPACE]",
```

Comment:
This is the usage string to be used in 
```go
Use:   "scale",
```
from above. Examples - in this case from `kubectl apply` - look like that:
```bash
Examples:                                                                                          
  # Apply the configuration in pod.json to a pod.       
  kubectl apply -f ./pod.json                                                                
                                                                                            
  # Apply resources from a directory containing kustomization.yaml - e.g. dir/kustomization.yaml.
  kubectl apply -k dir/                                                                         
                                  
  # Apply the JSON passed into stdin to a pod.                                               
  cat pod.json | kubectl apply -f -   
```